### PR TITLE
Updating Set-SPOTenant docs with new param details

### PR DESCRIPTION
**Pull request by devarma2**
Adding details for the new DisableOutlookPSTVersionTrimming parameter.

### CODE REVIEW

**Review by get-itips — COMMENTED** [review]
is this parameter going live in the next release? checked in 16.0.21213.12000 and it is not there yet, thanks for the clarification

**Comment by devarma2** [comment]
@get-itips This param should be in the version released last Friday (June 25), 16.0.21411.12000

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -1827,6 +1828,26 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisableOutlookPSTVersionTrimming
+
+As communicated in MC256835 (May ’21), starting August 16th 2021 the service will start retaining 30 days worth of versions for any PST files stored in OneDrive for Business and SharePoint Online team site document libraries. This change will prevent cases of previous versions of PST files quickly consuming available storage. The change only impacts previous versions of PST files stored in your document library storage. As best practice, PST files should not be uploaded on OneDrive for Business and SharePoint Online team site document libraries due to the impact on storage and network bandwidth.
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1833]:
```suggestion
As communicated in MC256835 (May ’21), starting August 16th 2021 the service will start retaining 30 days worth of versions for any PST files stored in OneDrive for Business and SharePoint Online team site document libraries. This change will prevent cases of previous versions of PST files quickly consuming available storage. The change only impacts previous versions of PST files stored in your document library storage. As a best practice, PST files should not be uploaded on OneDrive for Business and SharePoint Online team site document libraries due to the impact on storage and network bandwidth.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -1827,6 +1828,26 @@ Accept pipeline input: False
...
+
+If you wish to opt out of this policy change, you can use the parameter provided below. You will have until August 13th 2021 to opt out. This is a one time opt-out operation. If you don’t opt out by the deadline, the new behavior will automatically be enabled. The opt-out option for the cmdlet will have no effect after the deadline. Opt out only applies to existing document libraries and does not apply to new doclibs created after August 13th 2021.
```
Comment (dariomws) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:1835]:
```suggestion
If you wish to opt-out of this policy change, you can use the parameter provided below. You will have until August 13th 2021, to opt-out. This is a one time opt-out operation. If you don’t opt-out by the deadline, the new behavior will automatically be enabled. The opt-out option for the cmdlet will have no effect after the deadline. Opt-out only applies to existing document libraries and does not apply to new doc libs created after August 13th 2021.
```

**Review by dariomws — COMMENTED** [review]
Agree with @get-itips.

It seems that the last public version is 16.0.21213.12000:
https://www.powershellgallery.com/packages/Microsoft.Online.SharePoint.PowerShell/

**Comment by dariomws** [comment]
If you have a Microsoft alias, you can link your accounts in GitHub so that a blue "Microsoft submitter" label appears to any PR's or issue you submit. Guidance on it here: https://docs.opensource.microsoft.com/tools/github/index.html

Here is the site you can use to see the mappings between GitHub and Microsoft aliases: https://repos.opensource.microsoft.com/people?q=devarma2

**Comment by yogkumgit** [comment]
@devarma2 Please review and implement the changes. Thanks.

**Comment by devarma2** [comment]
This link has the latest build: https://www.microsoft.com/en-us/download/details.aspx?id=35588

From my understanding, the powershellgallery page likely won't be updated with the latest build until July ~12th, due to a delay.

**Comment by get-itips** [comment]
> This link has the latest build: https://www.microsoft.com/en-us/download/details.aspx?id=35588
> 
> From my understanding, the powershellgallery page likely won't be updated with the latest build until July ~12th, due to a delay.

Ok I could verify with that other link, thanks

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
Thank you @devarma2!

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Pull request merged**